Makefile

```diff
@@ -81,12 +79,14 @@ test-unit:
 
 .PHONY: test-e2e
 test-e2e: crds
-	operator-sdk --verbose test local ./test/e2e \
-		--up-local \
-		--namespace="$(TEST_NAMESPACE)" \
-		--go-test-flags="$(GO_TEST_FLAGS)" \
-		--local-operator-flags="$(OPERATOR_FLAGS)" \
-			$(OPERATOR_SDK_EXTRA_ARGS)
+	GO111MODULE=on ginkgo \
+		-nodes=8 \
+		-slowSpecThreshold=300 \
+		-randomizeAllSpecs \
+		-timeout=10m \
```

Comment:
Could we keep timeout configurable?

Reply:
Done, see https://github.com/redhat-developer/build/pull/163/commits/ed90050dabf13bf92dfa08603a06d35b6e04d06e.